### ISSUE TITLE
Packetbeat 0.5.0 RPM does not exist

### ISSUE THREAD

**Issue by bscott** [packetbeat/packetbeat-deploy#9]
Packetbeat 0.5.0 RPM does not exist in Github Releases Downloads

```
failed: [x.x.x.x] => {"changed": false, "failed": true, "rc": 1}
msg: Package at https://github.com/packetbeat/packetbeat/releases/download/v0.5.0/packetbeat-0.5.0-1.el6.x86_64.rpm could not be installed
```

```
https://github.com/packetbeat/packetbeat/releases
```

**Comment by bscott** [comment]
Setting this to 0.4.3 fixed my failure.

https://github.com/packetbeat/packetbeat-deploy/blob/master/roles/packetbeat/vars/main.yml#L2

**Comment by bscott** [comment]
@packetbeat

**Comment by tsg** [comment]
Right, it's because packetbeat-deploy got a little ahead of packetbeat, which doesn't yet have a 0.5.0 version. I just a pushed a v0.4.3 tag to packetbeat-deploy, so another temporary workaround would be to checkout this tag before running the ansible commands.

**Comment by tsg** [comment]
Fixed since we've released 0.5.0.
